### PR TITLE
add_issue_template.md: Create issue templates task

### PR DESCRIPTION
**Pull request by Monal5031**
Created repetable issue templates
task for GCI on projects page

Closes https://github.com/coala/projects/issues/408

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by prashant0598** [comment]
ack 26486ba

File: _projects/add_issue_template.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+Check that the repository does not have a `ISSUE_TEMPLATE.md` file.
+Maybe GitHub does not recognise the contents of the file.
+
+Check the pull requests for the repository.
+Maybe someone has already proposed a `ISSUE_TEMPLATE.md` file.
+Do not send another pull request to add a `ISSUE_TEMPLATE.md` file.
+
+## Step 2
+
+### Does the repository already have a issue template selected?
```
Comment (neelanshsahai) [inline _projects/add_issue_template.md]: an* issue template

File: _projects/add_issue_template.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Step 3
+
+You can read more about the standard templates [here](https://github.com/blog/2111-issue-and-pull-request-templates)
+
+1. Copy the templates and/or points in a file temporarily.
+2. Fork the repository
+3. Add a `ISSUE_TEMPLATE.md` file to the repository, committing the new file to a branch.
+   1. Make sure you paste all the contents properly in the file.
+   2. Make sure you don't miss out on any points which need to be in the file.
```
Comment (neelanshsahai) [inline _projects/add_issue_template.md]: miss out* any points which must be there* in the file.

**Comment by neelanshsahai** [comment]
unack https://github.com/coala/projects/commit/26486ba51537741ecc9bff08ee84f3f4dcb44229

**Comment by prashant0598** [comment]
ack 4775bb2

**Comment by jayvdb** [comment]
Could you add this to the private task document instead.
(And we'll re-open this near the launch date.)

**Pull request closed without merging**